src/InfoBarWidget.js

```diff
@@ -0,0 +1,122 @@
+var InfoBarWidget = function InfoBarWidget( config ) {
+	var widget = this;
```

Comment:
do we have ES6+ here? Because we should be using `let` or `const` (`const` in both lines above, I think)

Reply:
Or maybe we don't need `widget` at all, see below